### PR TITLE
nits

### PR DESCRIPTION
**Pull request by y-z11**
* Add examples
* Rename flag 'gasUser' to 'gas_user'
* Move elapsed time in Execute() (or it will output the elapsed time even after help msgs)

### CODE REVIEW

File: iwallet/transfer.go [hunk 1]
```diff
@@ -25,7 +25,8 @@ var transferCmd = &cobra.Command{
 	Aliases: []string{"trans"},
 	Short:   "Transfer IOST",
 	Long:    `Transfer IOST`,
-	Example: `  iwallet transfer test1 100 --account test0`,
+	Example: `  iwallet transfer test1 100 --account test0
+  iwallet transfer test1 100 --account test0 --memo "just for test :D\n中文测试\n😏"`,
```
Comment (y-z11) [inline iwallet/transfer.go]: Is unicode/emoji valid?

Comment (lispc) [reply iwallet/transfer.go]: 所以能吗。。？

Comment (y-z11) [reply iwallet/transfer.go]: 看起来可以。。。
image

Comment (lispc) [reply iwallet/transfer.go]: orz

**Pull request merged**